setup.py

```diff
@@ -15,7 +15,7 @@ def parse_requirements():
 
 setup(
     name='polyswarm-api',
-    version='0.5.0',
+    version='0.5.1',
```

Comment:
you can just read this from the `_version.py` so you don't have to change it in 2 places.

Reply:
When I first added version, I considered doing this, but it's really hacky and would prefer to just do this.